test/AudioPlayr/addEventListener.ts

```diff
@@ -0,0 +1,39 @@
+/// <reference path="../../node_modules/@types/chai/index.d.ts" />
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
+/// <reference path="../../lib/AudioPlayr.d.ts" />
+/// <reference path="../utils/MochaLoader.ts" />
+/// <reference path="../utils/mocks.ts" />
+
+mochaLoader.addTest("does not call the callback before the sound is played", (done): void => {
+    // Arrange
+    const AudioPlayer = mocks.mockAudioPlayr();
+    const sound = AudioPlayer.library[mocks.mockSoundName];
```

Comment:
`.library` is private. It's unfortunate that `mocks` is typed as `any` right now, but we should pretend it's getting typed correctly.